CMakeLists.txt

```diff
@@ -14,11 +14,7 @@
 # limitations under the License.
 # ~~~
 
-cmake_minimum_required(VERSION 3.10)
-
-if (NOT (CMAKE_VERSION VERSION_LESS "3.14"))
-    cmake_policy(SET CMP0091 NEW)
```

Comment:
Just pointing out that this was deleted, in case the delete was unintentional. I don't really understand what it was doing to begin with.

https://cmake.org/cmake/help/latest/policy/CMP0091.html

something or other about `CMAKE_MSVC_RUNTIME_LIBRARY` which we do use:

https://github.com/GoogleCloudPlatform/functions-framework-cpp/blob/0279cddf0d879e8ea7b13ade2b0431512a641724/CMakeLists.txt#L47-L51

Reply:
It was intentional.  `cmake_minimum_required(VERSION 3.10...3.26)` implicitly bumps the `CMP00091` to the new behavior if it is available.